main.js

```diff
@@ -0,0 +1,74 @@
+import {
+  renderBooks,
+  titleInput,
+  authorInput,
+  addBook,
+  localBooks,
+} from './modules/variables.js';
+
+// import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
```

Comment:
- [ ] In order to keep your codebase clean please remove any commented out code.